### PR TITLE
Add random name to main page

### PR DESCRIPTION
**Pull request by ak47na**
Update DataServlet.java to send back to the main page a random name:

getRandomName() in script.js is called when a user clicks: "Fetch a name!" button
The name is fetched from the server using DataServlet.

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -14,7 +14,8 @@
 
 package com.google.sps.servlets;
 
-import java.io.IOException;
+import java.io.*;//IOException;
```
Comment (LiboShen) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Remove the commented code?

File: portfolio/src/main/webapp/placesGallery.html [hunk 2]
```diff
@@ -15,4 +15,4 @@ <h1>Favourite Places Gallery</h1>
       </div>
     </div>
   </body>
-</html>
+</html>
```
Comment (LiboShen) [inline portfolio/src/main/webapp/placesGallery.html]: Accidentally changed the file? Please revert it :)

**Review by LiboShen — CHANGES_REQUESTED** [review]

**Comment by LiboShen** [comment]
> Update DataServlet.java to send back to the main page a random name:
> 
> getRandomName() in script.js is called when a user clicks: "Fetch a name!" button
> The name is fetched from the server using DataServlet.

Ah, I forgot to mention, it's great you add some detailed description of your PR. Good job!

**Comment by LiboShen** [comment]
Hi Ana, did you forgot to push your new commit?

**Review by LiboShen — APPROVED** [review]
Nice!

**Comment by ak47na** [comment]
> Hi Ana, did you forgot to push your new commit?

I pushed the new commit, but I'm not sure why it doesn't appear in the commit list of the pull request.

**Pull request merged**